sale.xml

```diff
@@ -16,7 +16,18 @@
             <field name="type">form</field>
             <field name="name">sale_payment_add_form</field>
         </record>
-
+        <record model="ir.action.act_window" id="act_sale_relate">
+            <field name="name">Payments</field>
+            <field name="res_model">sale.payment</field>
+            <field name="domain">[
```

Comment:
should be in one line: `<field name="domain">[('sale', 'in', Eval('active_ids'))]</field>`